lib/sqlitex/row.ex

```diff
@@ -56,8 +56,10 @@ defmodule Sqlitex.Row do
     {String.to_integer(yr), String.to_integer(mo), String.to_integer(da)}
   end
 
-  defp to_time(time) do
-    <<hr::binary-size(2), ":", mi::binary-size(2), ":", se::binary-size(2), ".", fr::binary-size(6)>> = time
+  defp to_time(<<hr::binary-size(2), ":", mi::binary-size(2), ":", se::binary-size(2), ".", fr::binary-size(6)>>) do
```

Comment:
If we are worried about the number of decimals in `fr`, maybe we could just capture the last characters:

```
<<hr::binary-size(2), ":", mi::binary-size(2), ":", se::binary-size(2), ".", fr::binary>>
```

without specifying the size.  Thoughts?